src/overmap_connection.cpp

```diff
@@ -85,6 +89,12 @@ void overmap_connection::subtype::deserialize( JsonIn &jsin )
     load( jo );
 }
 
+void overmap_connection::clear_subtype_cache() const
+{
+    std::vector<cache> newcache = std::vector<cache>( overmap_terrains::get_all().size() );
```

Comment:
```suggestion
    auto newcache = std::vector<cache>( overmap_terrains::get_all().size() );
```
[nitpick] no need to repeat types